### PR TITLE
[4.1] 'return' is unnecessary as the last statement in a method

### PR DESCRIPTION
**Pull request by PhilETaylor**
code review

### CODE REVIEW

File: administrator/components/com_scheduler/src/Task/TaskOption.php [hunk 1]
```diff
@@ -86,6 +86,5 @@ public function __get(string $name)
 			return $this->$name;
 		}
 
-		return;
```
Comment (Fedik) [inline administrator/components/com_scheduler/src/Task/TaskOption.php:89]: maybe better would be `return null;`  because it is `__get`

Comment (PhilETaylor) [reply administrator/components/com_scheduler/src/Task/TaskOption.php:89]: No. They both would return null as evidenced here https://3v4l.org/YqP4K

Comment (Fedik) [reply administrator/components/com_scheduler/src/Task/TaskOption.php:89]: it still better to have `return null`, 
check doc block: "return mixed"

**Comment by HLeithner** [comment]
please remove the blank line between the }

File: administrator/components/com_scheduler/src/Task/TaskOption.php (hunk 1 above)
Comment (PhilETaylor) [reply administrator/components/com_scheduler/src/Task/TaskOption.php:89]: disagree - so does an IDE and Psalm - but hey - have it your way, null added.

**Comment by PhilETaylor** [comment]
> please remove the blank line between the }

No I just put it back in as @Fedik wants to return again

File: administrator/components/com_scheduler/src/Task/TaskOption.php (hunk 1 above)
Comment (Fedik) [reply administrator/components/com_scheduler/src/Task/TaskOption.php:89]: thanks :)

**Comment by Fedik** [comment]
I have tested this item :white_check_mark: successfully on fab6c5f0ad0432134fc780e09b5bc8ca5ea207ef<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/36128">issues.joomla.org/tracker/joomla-cms/36128</a>.</sub>

**Comment by HLeithner** [comment]
now we have 2 tabs instead of an empty line, I think thats wrong ;-)

**Comment by PhilETaylor** [comment]
blame GitHub online editing :(

**Comment by HLeithner** [comment]
thanks

**Pull request merged**